bin/cheribsdtest/cheribsdtest_vm.c

```diff
@@ -1030,4 +1031,74 @@ CHERIBSDTEST(cheribsdtest_vm_reservation_mmap_fixed_insert_noperm,
 
 	cheribsdtest_success();
 }
-#endif
+
+#if PMAP_HAS_LARGEPAGES
+static int
+get_pagesizes(size_t ps[MAXPAGESIZES])
```

Comment:
```suggestion
get_pagesizes(size_t ps[static MAXPAGESIZES])
```
otherwise the explicit size isn't actually doing anything